### ISSUE TITLE
[med] Color Contrast: Many Pages: Grey icons against white background fail contrast requirements.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8429]
## Action Performed:
1. Using Chrome, open URL staging.new.expensify.com
2. Select (+) button.
3. After expanding the + button select new group control.
4. New group window is appearing, Notice close (x) button.
5. Measure the color contrast ratio of the grey icons against white background for close (x) button.

## Expected Result:
The visual presentation of meaningful graphical objects or interactive components should have a contrast ratio of at least 3.00:1 with the surrounding background color.

## Actual Result:
The grey icons (#C6C9CA) on white background (#FFFFFF) have a color contrast ratio of 1.7:1.

**Other occurrences**
Same issue also repro on PR #8809

## Workaround:
No

## Area issue was found in:
New group window and all pages of the expensify where close (x) button is appearing.

## Failed WCAG checkpoints
1.4.11

## User impact:
The low-vision and color-blind users might not be able to see the icons.

## Suggested resolution:
Use a darker shade of grey for the icons to achieve a color contrast ratio of at least 3.00:1. One example of color to use is #8D9596, which will give a color contrast ratio of 3.05:1.

## Platform:
- Web
- Android
- Mobile Web

**Version Number:**v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

![7530_CCa fail for X button](https://user-images.githubusercontent.com/97600950/161194450-d9b17ff8-cf0d-429a-991d-f7de7d63f2b0.png)
